### PR TITLE
 [docs] PR for Flows in Platform for Remove/Release/Delete nodes (Doc10332) 

### PR DESCRIPTION
**Pull request by lizayugabyte**
Preview: https://deploy-preview-10871--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/manage-deployments/remove-nodes/

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md [hunk 1]
```diff
@@ -14,20 +14,74 @@ isTocNested: true
...
+- Stopped
+- ToBeRemoved
+
+Taking this action transfers the node to a Removing and then Removed state, as follows:
+
+1. If the node is a Master leader, it is removed (the Master process is stopped and there is a wait for the new Master leader).
+2. The TServer is marked as blacklisted on the Master leader.
+3. There is a wait for tablet quorums to remove the blacklisted TServer.
+4. Data migration is performed and the TServer process stops only if it is reachable.
+5. The node is marked as having neighter Master nor TServer processes.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md:43]: neighter

File: docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md [hunk 2]
```diff
@@ -14,20 +14,74 @@ isTocNested: true
...
-So, for the `yb-15-node-actions-n3` node, you can use the **Actions** drop-down list at the end and choose the **Remove Node** option. This will bring the node to the `Removed` status and no further data can be accessed from this node as server processes are shut down.
+When this happens, new Master leaders are elected for the underlying data shards, but since the universe enters a partially under-replicated state, it would not be able to tolerate additional failures. To remedy the situation, the unreachable node needs to be eliminated by taking actions in the following sequence:
+
+- Step 1: [Remove node](#remove-node)
+- Step 2: [Release instance](#release-instance)
+- Step 3: [Delete node](delete-node)
+
+## Remove node
+
+The action to remove a node is available from the following node states:
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md:29]: Is state the same as status? Should 'Unreachable' be in this list?

**Review by ddhodge — APPROVED** [review]
LGTM; typo, suggestion/question

File: docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md:43]: Fixed

**Review by SergeyPotachev — APPROVED** [review]
Looks good to me. About the `Unreachable` state - it is quite complex question. It requires a separate description/discussion.

File: docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md:29]: Good catch, I'm going to check this

Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md]: I've got the answer of this and modified the doc

**Pull request merged**